packages/ERTP/src/amountMath.js

```diff
@@ -38,44 +51,46 @@ import './types';
  * function `coerce` takes an amount and checks it, returning an amount (amount
  * -> amount).
  *
- * `makeAmount` takes in a brand and the name of the particular
- * mathHelpers to use.
+ * `makeAmountMath` takes in a brand and the kind of mathHelpers to use.
  *
  * amountMath is unfortunately not pass-by-copy. If you call
- * `getAmountMath` on a remote issuer, it will be a remote object and
+ * `getAmountMath` on a remote issuer, it will return a remote object and
```

Comment:
There's no `getAmountMath` anymore, right?

Reply:
yep.  I re-wrote this paragraph.